src/atc/delta/deltastream_handle.py

```diff
@@ -0,0 +1,239 @@
+from typing import List
+
+from pyspark.sql import DataFrame
+from pyspark.sql.streaming import DataStreamWriter
+
+from atc.configurator.configurator import Configurator
+from atc.functions import init_dbutils
+from atc.spark import Spark
+from atc.tables.SparkHandle import SparkHandle
+from atc.utils import GetMergeStatement
+from atc.utils.FileExists import file_exists
+
+
+class DeltaStreamHandle(SparkHandle):
+    def __init__(
+        self,
+        *,
+        checkpoint_path: str,
+        name: str = None,
+        data_format: str = None,
+        location: str = None,
+        trigger_type: str = "availablenow",
+        trigger_time: str = None,
+        await_termination=False,
```

Comment:
remove await_termination from here?

Reply:
This should might not be controlled by the configurator  but should perhapse be controlled by a class method.